README.rst

```diff
@@ -58,15 +57,15 @@ Demos & Examples
 ================
 
 In the `example section <https://mathurinm.github.io/celer/auto_examples/index.html>`__ of the documentation,
-you will find numerous examples on real life datasets,
-timing comparison with other estimators, easy and fast ways to perform cross validation, etc.
+you can find numerous examples on real-life datasets,
+timing comparison with other estimators as well as easy and fast ways to perform cross-validation.
 
 
 Dependencies
 ============
 
 All dependencies are in the ``./requirements.txt`` file.
-They are installed automatically when ``pip install -e .`` is run.
+They are installed automatically when running ``pip install -e .``.
```

Comment:
Aren't both correct? If there is no clear advantage avoid these kind of changes which introduce noise in the PR

Reply:
There are equivalent. However, I found the latter more smooth when reading it.

Reply:
Ok for this change.
In general: try to avoid changes when the benefit is not obvious. If it is not broken don't fix it.